coriolis/conductor/rpc/server.py

```diff
@@ -924,19 +967,18 @@ def _handle_post_task_actions(self, ctxt, task, execution, task_info):
             # Replica instance as well.
             volumes_info = task_info.get("volumes_info")
             if volumes_info:
-                self._update_volumes_info_for_migration_parent_replica(
-                    ctxt, execution.action_id, task.instance,
-                    {"volumes_info": volumes_info})
+                self._update_volumes_info(
+                    ctxt, task.instance, execution, volumes_info)
 
         elif task_type == constants.TASK_TYPE_DELETE_REPLICA_DISK_SNAPSHOTS:
-
-            if not task_info.get("clone_disks"):
+            # If this execution is a replica, volumes_info should NOT be
+            # deleted.
+            clone_disks = task_info.get("clone_disks", True)
```

Comment:
1) Any real reason to declare this variable separately considering it's only used once?
2) There is no reason why 'backend code' should make assumptions about params which imply extra work (such as defaulting 'clone_disks' to True here).
This is already done in the API-layer (where it should be), so please remove the default here.